src/App.tsx

```diff
@@ -47,20 +47,15 @@ export class App extends React.Component<{}, State> {
           {selectedGood
             ? `${selectedGood} is selected`
             : 'No goods selected'}
-          <button
-            type="button"
-            onClick={() => this.removeGood()}
-          >
-            X
-          </button>
```

Comment:
Why did you delete this button?
You need to render it when you have selected good (if not - don't render)